contracts/TrueReward/mocks/ATokenMock.sol

```diff
@@ -1,10 +1,11 @@
-pragma solidity 0.5.13;
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
 
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import { ERC20Mock } from "./ERC20Mock.sol";
```

Comment:
How is the mock different than the actual ERC-20?

Reply:
openzeppelin ERC20 doesn't have virtual balanceOf, otherwise they are same